src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -212,22 +212,19 @@ class BinaryCompactObject : public DomainCreator<3> {
     using type = bool;
     static constexpr Options::String help = {
         "Use equiangular instead of equidistant coordinates."};
-    static type default_value() { return false; }
```

Comment:
I would keep this option (at least as the suggested value) because currently equiangular does not work for BinaryCompactObject and there is no current test that indicates that it doesn't work. I hope to have that fixed soon... :disappointed: